src/index.js

```diff
@@ -20,31 +18,16 @@ const tasks = [
   },
 ];
 
-window.onload = () => {
+// document.querySelector('resync-icon').src = Icon;
```

Comment:
Please remove these commented pieces of code. Remember that keeping your codebase clean can improve tremendously the maintainability and readability of your project! 😎🤓